src/sentry/api/endpoints/project_details.py

```diff
@@ -68,13 +71,26 @@ def clean_newline_inputs(value, case_insensitive=True):
     return result
 
 
-class ProjectMemberSerializer(serializers.Serializer):
+class AbstractProjectSerializer(serializers.Serializer):
```

Comment:
I am wondering what was the best way to give those two classes mutual functionality. Please advise.